x-pack/plugins/remote_clusters/public/application/sections/remote_cluster_list/components/security_model/security_model.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiText, EuiFlexGroup, EuiFlexItem, EuiIconTip } from '@elastic/eui';
+
+import { getSecurityModel } from '../../../../../../common/constants';
+import { Cluster } from '../../../../../../common/lib/cluster_serialization';
+
+export function SecurityModel({ securityModel }: { securityModel: Cluster['securityModel'] }) {
+  return (
+    <EuiFlexGroup gutterSize="s" alignItems="center">
+      <EuiFlexItem grow={false}>
+        <EuiText data-test-subj="authenticationTypeLabel" size="s">
+          {getSecurityModel(securityModel)}
+        </EuiText>
+      </EuiFlexItem>
+
+      {securityModel !== 'api_key' && (
+        <EuiFlexItem grow={false} data-test-subj="authenticationTypeWarning">
+          <EuiIconTip
+            type="iInCircle"
+            color="subdued"
+            content={
+              <FormattedMessage
+                id="xpack.remoteClusters.remoteClusterList.table.authTypeWarningMessage"
+                defaultMessage="There is a new authentication type available. Create a new remote cluster to use it."
```

Comment:
```suggestion
                defaultMessage="If you need more fine-grained permissions on your remote cluster, you can now use API keys instead of certificates as authentication mechanism. Create a new remote cluster to use it."
```
Suggestion to be more explicit on what is this new authentication type (API key), as well as why it should be considered (fine-grained permissions). Let me know what you think @sabarasaba

Reply:
👍 We should keep it as suggested above https://github.com/elastic/kibana/pull/173489#discussion_r1431078929

Reply:
I ended up removing it because I spoke to the ES team, and they confirmed that its not necessary to remove it since it can be changed with an existing one without having to create a new one.